pkg/runner/test_runner.go

```diff
@@ -197,19 +198,11 @@ func (tr *TestRunner) Run(cancel, pause <-chan struct{}, test *test.Test, target
 	// the pipeline.
 	for {
 		select {
-		case <-cancel:
```

Comment:
We do not need to wait for cancel/pause. If this event occures, testPipeline.run is expected to terminate and will write something to errCh